src/pages/workspace/accounting/qbo/advanced/QuickbooksAdvancedPage.tsx

```diff
@@ -75,6 +103,24 @@ function QuickbooksAdvancedPage({policy}: WithPolicyProps) {
             pendingAction: pendingFields?.autoCreateVendor,
             errors: ErrorUtils.getLatestErrorField(qboConfig ?? {}, CONST.QUICK_BOOKS_CONFIG.AUTO_CREATE_VENDOR),
             onCloseError: () => Policy.clearQBOErrorField(policyID, CONST.QUICK_BOOKS_CONFIG.AUTO_CREATE_VENDOR),
+            wrapperStyle: styles.mv3,
+        },
+        {
+            title: translate('workspace.qbo.advancedConfig.reimbursedReports'),
+            subtitle: translate('workspace.qbo.advancedConfig.reimbursedReportsDescription'),
+            isActive: isSyncReimbursedSwitchOn,
+            onToggle: () =>
+                Connections.updatePolicyConnectionConfig(
+                    policyID,
+                    CONST.POLICY.CONNECTIONS.NAME.QBO,
+                    CONST.QUICK_BOOKS_CONFIG.COLLECTION_ACCOUNT_ID,
+                    isSyncReimbursedSwitchOn ? '' : [...qboAccountOptions, ...invoiceAccountCollectionOptions][0].id,
```

Comment:
@teneeto  Shouldn't this be based on the `invoiceAccountCollectionOptions` only? This switch will set a collect account and `qboAccountOptions` may not have valid options (e.g. credit cards)